src/plugins/console/server/lib/spec_definitions/json/generated/ml.validate.json

```diff
@@ -1,11 +1,21 @@
 {
   "ml.validate": {
+    "url_params": {
+      "error_trace": "__flag__",
+      "filter_path": [],
+      "human": "__flag__",
+      "pretty": "__flag__"
+    },
     "methods": [
       "POST"
     ],
     "patterns": [
       "_ml/anomaly_detectors/_validate"
     ],
-    "documentation": "https://www.elastic.co/guide/en/machine-learning/current/ml-jobs.html"
+    "documentation": "https://www.elastic.co/guide/en/machine-learning/current/ml-jobs.html",
+    "availability": {
```

Comment:
What does it mean if the API is `false` in both environments?

Reply:
Usually that is either an internal endpoint like for example [`_internal.delete_desired_balance`](https://www.elastic.co/guide/en/elasticsearch/reference/current/delete-desired-balance.html), or it's still under a feature flag, or in this case it looks like it's a deprecated/moved/renamed endpoint. I was not able to find this one in the docs pages ([link](https://www.elastic.co/guide/en/machine-learning/current/ml-jobs.html)).

Reply:
Thanks for the explanation. It sounds like this one may also need to be updated in the specs then?

Reply:
Yes, I think this one is probably moved to a different endpoint or deprecated completely. This should be updated in the spec.